index.js

```diff
@@ -9,9 +9,19 @@ module.exports = function WebpackHotServer(compiler, options) {
     app.use(require('webpack-hot-middleware')(compiler))
   }
 
+  if (options.proxy) {
+    var proxy = require('express-http-proxy')
+    Object.keys(options.proxy).map(function (path) {
+  		app.use(path, proxy(options.proxy[path], {
+        forwardPath: function(req, res) {
+          return require('url').parse(req.url).path;
+        }
+      }));
+  	});
```

Comment:
Hate to be nitpicky, but can you please match the code style with the rest of the document? (no semicolons, and fix the indentation?)